js/bundle.go

```diff
@@ -246,7 +246,7 @@ func (b *Bundle) Instantiate(logger logrus.FieldLogger, vuID int64) (bi *BundleI
 		Runtime: rt,
 		Context: ctxPtr,
 		exports: make(map[string]goja.Callable),
-		env:     b.RuntimeOptions.Env,
+		env:     rt.Get("__ENV").Export().(map[string]string),
```

Comment:
It will be better if we don't have to cache it back and again ... but for that I will need to get it somehow out of `b.instantiate`

which will be a lot bigger change IMO.

Given that we set it *always* there is no chance for this to panic unless someone changes what `__ENV` is ... :(  Which should probably not be possible, to begin with.

That will probably require making it nonconfigurable/writable and then only changing the object - again a lot bigger change, but arguably way more accurate.

Maybe I should just check that it is `map[string]string` and set it instead of `b.RuntimeOptions.Env` for this hotfix?